packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -154,10 +155,7 @@ export class SamplerOperations {
         this.poolsCaches = poolsCaches
             ? poolsCaches
             : {
-                  [ERC20BridgeSource.Beethovenx]: new BalancerV2PoolsCache(
-                      chainId,
-                      BEETHOVEN_X_SUBGRAPH_URL_BY_CHAIN[chainId],
-                  ),
+                  [ERC20BridgeSource.Beethovenx]: BalancerV2PoolsCache.createBeethovenXPoolCache(chainId),
```

Comment:
So only `Beethovenx` uses `BalancerV2PoolsCache` now? Since `BalancerV2` uses the newer `BalancerV2SwapInfoCache`? If that is the case I'd ultimately recommend a rename.

Reply:
Yes, that's the case!

I will rename in a follow up PR.